templates/header/topbar_end.html

```diff
@@ -67,6 +67,19 @@
                     <input id="nav-search" name="query" type="text" aria-label="Find crate by search query" tabindex="-1"
                         placeholder="Find crate" {%- if search_query %} value="{{ search_query }}" {%- endif %}>
                 </div>
+                {# The Search Sort by dropdown menu#}
```

Comment:
That will very likely mess up the media queries since it'll change the size of the content in the topbar.

Reply:
em, I test it on Edge web browser, the width and height of the select-box  stay the same when i change the option.

Is there anything else I need to test.

Reply:
Sorry, that's not what I meant: reduce the width of your window. Normally, the topbar icons should never be on two lines.

Reply:
So you mean: The new drop-down box is not recommended to be placed at the topbar ?

Reply:
I don't think it's a good idea indeed, as the topbar is already very loaded and also because this box should be closer to the element it interacts with.

Reply:
I'll try to move the drop-down box to the top-right corner of the search results, as crates.io did. But I ran into some problems (some code issues), maybe I will come out with another version next week.